packages/web-components/src/dropdown/dropdown.styles.ts

```diff
@@ -0,0 +1,270 @@
+  colorCompoundBrandStroke,
+  colorNeutralBackground1,
+  colorNeutralBackground3,
+  colorNeutralBackgroundDisabled,
+  colorNeutralForeground1,
+  colorNeutralForeground3,
+  colorNeutralForeground4,
+  colorNeutralForegroundDisabled,
+  colorNeutralStroke1,
+  colorNeutralStroke1Hover,
+  colorNeutralStroke1Pressed,
+  colorNeutralStrokeAccessible,
+  colorNeutralStrokeAccessibleHover,
+  colorNeutralStrokeAccessiblePressed,
+  colorNeutralStrokeDisabled,
+  colorTransparentBackground,
+  colorTransparentStroke,
+  colorTransparentStrokeInteractive,
+  curveAccelerateMid,
+  curveDecelerateMid,
+  durationNormal,
+  durationUltraFast,
+  lineHeightBase200,
+  lineHeightBase300,
+  lineHeightBase400,
+  spacingHorizontalM,
+  spacingHorizontalMNudge,
+  spacingHorizontalS,
+  spacingHorizontalSNudge,
+  spacingHorizontalXXS,
+  spacingVerticalS,
+  spacingVerticalSNudge,
+  spacingVerticalXS,
+  strokeWidthThick,
+  strokeWidthThin,
+} from '../theme/design-tokens.js';
+import { display } from '../utils/display.js';
+
+/**
+ * Styles for the {@link (Dropdown:class)} component.
+ *
+ * @public
+ */
+export const styles = css`
+  ${display('inline-flex')}
+
+  :host {
+    anchor-name: --dropdown-trigger;
+    box-sizing: border-box;
+    color: ${colorNeutralForeground1};
+    cursor: pointer;
+  }
+
+  :host(${placeholderShownState}) {
+    color: ${colorNeutralForeground4};
+  }
+
+  .control {
+    align-items: center;
+    appearance: none;
+    background-color: ${colorNeutralBackground1};
+    border-radius: ${borderRadiusMedium};
+    border: none;
+    box-shadow: inset 0 0 0 ${strokeWidthThin} var(--control-border-color);
+    box-sizing: border-box;
+    color: inherit;
+    column-gap: ${spacingHorizontalXXS};
+    display: inline-flex;
+    justify-content: space-between;
+    min-width: 160px;
+    overflow: hidden;
+    padding: ${spacingVerticalSNudge} ${spacingHorizontalMNudge};
+    position: relative;
+    text-align: start;
+    width: 100%;
+    z-index: 1;
+    ${typographyBody1Styles}
+  }
+
+  :host(${smallState}) .control {
+    column-gap: ${spacingHorizontalXXS};
+    padding: ${spacingVerticalXS} ${spacingHorizontalSNudge};
+    ${typographyCaption1Styles}
+  }
+
+  :host(${largeState}) .control {
+    column-gap: ${spacingHorizontalS};
+    padding: ${spacingVerticalS} ${spacingHorizontalM};
+    ${typographyBody2Styles}
+  }
+
+  ::slotted(:is(input, button)) {
+    all: unset;
+    flex: 1;
+  }
+
+  ::slotted(button) {
+    cursor: pointer;
+  }
+
+  ::slotted(input) {
+    cursor: text;
+  }
+
+  ::slotted([slot='indicator']) {
+    all: unset;
+    align-items: center;
+    appearance: none;
+    aspect-ratio: 1;
+    color: ${colorNeutralForeground3};
+    display: inline-flex;
+    justify-content: center;
+    width: 20px;
+  }
+
+  :host(${smallState}) ::slotted([slot='indicator']) {
+    width: 16px;
+  }
+
+  :host(${largeState}) ::slotted([slot='indicator']) {
+    width: 24px;
```

Comment:
I assume fluent does not have tokens for these sizes... It seems like it should...

Reply:
AFAIK this is correct. Size values without tokens are present in other components, including avatar, button, and badge (and probably more).